cartridges/int_cloudinary_sfra/cartridge/controllers/Tile.js

```diff
@@ -26,7 +26,7 @@ server.append('Show', cache.applyPromotionSensitiveCache, function (req, res, ne
                 cloudinaryModel.addCloudinaryProductImage(product, productPrimaryImg);
             }
 
-            if (!empty(productPrimaryImg.altText)) {
+            if (!empty(productPrimaryImg) && !empty(productPrimaryImg.altText)) {
```

Comment:
please use standard notation: productPrimaryImg && (productPrimaryImg.altText || productPrimaryImg.altText === "")
so we make sure we allow emptying the at text

Reply:
@asad-rafter Please address my concern